roles/splunk/tasks/configure_auditd.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "get {{ splunk_nix_user }} user uid"
+  getent:
+    database: passwd
+    key: "{{ splunk_nix_user }}"
+
+# - debug:
```

Comment:
Please remove this and other commented out lines

Reply:
I'll take care of this in the next commit.